source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.ts

```diff
@@ -8,6 +8,13 @@ export default {
 
   updated: 'Updated!',
 
+  opensource: 'Open Source',
```

Comment:
few missing keys in es-pe, need to put it even though they are not translated

Reply:
I don't think so, it should fallback to english?

Reply:
it will fall back, but still translation need right?

Reply:
it falls back, so we don't need translation :)